Marlin/src/lcd/dogm/status_screen_DOGM.h

```diff
@@ -468,7 +470,7 @@ static void lcd_impl_status_screen_0() {
   // Status line
   //
 
-  #define STATUS_BASELINE (55 + INFO_FONT_HEIGHT)
+  #define STATUS_BASELINE (53 + INFO_FONT_HEIGHT)
```

Comment:
Now I'm wondering why this value isn't closer to 60, since the screen is 64 pixels tall.